app/src/main/java/org/greenstand/android/TreeTracker/activities/ImageCaptureActivity.kt

```diff
@@ -118,6 +118,11 @@ class ImageCaptureActivity : AppCompatActivity() {
         CameraX.bindToLifecycle(this, preview, imageCapture)
     }
 
+    override fun onBackPressed() {
+        super.onBackPressed()
+        deviceOrientation.disable()
```

Comment:
This is to avoid the scenario where device orientation data is continued to be captured when a back button is pressed from within the ImageCaptureActivity.

Reply:
It is unrelated here but found this while testing the app for this feature.  A minor fix and didn't want to raise a separate issue for resolving this.